### DISCUSSION TITLE
Clone Command

### DISCUSSION THREAD

**Discussion by cburch824** [lindell/multi-gitter#234] [General]
Hi,

I needed to clone a large number of repos from org, crawl those files for specific information, and generate a dashboard for an org's repos. I added a clone command to multi-gitter on my fork for the first part of that task:
https://github.com/lindell/multi-gitter/compare/master...cburch824:master

In short, the clone command preserves the org/group/repo structure when repos are cloned down: 

.
├── company1                    # Org / Top Level Group
│   ├── repo1                     # repo1
│   ├── repo2                     # repo2
│   └── group1                   # Group 1
│      └── repo3                  # repo3 under Group 1
└── ...

Would this feature be a potential candidate for a PR into the main repo?

**Comment by cburch824** [comment]
Hi @lindell, can you take a look when you have time? Thanks!

**Comment by lindell** [comment]
I see many people needing to use this. So a command might be useful. The only counter argument to adding this is that you could do it with the print command and a one line script already.

```
#!/bin/bash
cp . /path/to/root/$REPOSITORY
```

But it might not be that discoverable which is a counter argument to the counter argument.

**Reply by cburch824** [reply]
Thank you for taking the time to think it through @lindell . That's clever, I liked that script suggestion and ran with it. I believe the simplicity of the changes justify using a script, rather than maintaining a new command. I added PR #237 as a result. 

I still have the code for the clone command in my master branch if you've thought about it more and would rather go that route, but I personally think the script route turned out quite well.
